jwst/jump/jump_step.py

```diff
@@ -29,9 +29,19 @@ class JumpStep(Step):
         min_sat_area = float(default=1.0) # minimum required area for the central saturation of snowballs
         min_jump_area = float(default=5.0) # minimum area to trigger large events processing
         expand_factor = float(default=2.0) # The expansion factor for the enclosing circles or ellipses
-        use_ellipses = boolean(default=False) # Use an enclosing ellipse rather than a circle for MIRI showers
+        use_ellipses = boolean(default=False) # deprecated
```

Comment:
This should be removed from the code. If it no longer has any function in the code, it shouldn't be in the arg list.

Reply:
OK, now that I understand that this was intentionally left in there to support backwards compatibility, I withdraw my comment.